ONSdigital/rm-action-service
# Added action state diagram and a Makefile to build

# Motivation and Context
This PR adds plantuml state diagram for action and the means to build it

# What has changed
No code has been changed but a Makefile has been added

# How to test?
```
make diagrams
/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome diagrams/action-state.svg
```
Ensure diagrams can be viewed
